encoding.bs

```diff
@@ -879,7 +879,7 @@ fallback encoding <var>encoding</var>, run these steps:
 <ol>
  <li><p>Let <var>buffer</var> be an empty byte sequence.
 
- <li><p>Let <var>BOM seen flag</var> be unset.
+ <li><p>Let <var>BOM seen</var> be false.
```

Comment:
Can we make this `BOMSeen` while changing this? These days the convention seems to be to not use spaces for local variables.